Login + Sign up modals fixed

MariCerv opened the pull request:
- new js file to fix inputs
- font colors fixed
- github log in fixed

MariCerv committed:
Login + Sign up modals fixed

- new js file to fix inputs
- font colors fixed
- github log in fixed

MariCerv committed:
Indentation fixed

dethos commented on hawkpost/templates/account/auth_modals.html:
```diff
@@ -98,15 +88,27 @@ <h3 class="title"> Sign up</h3>
 </div>
 <div class="md-overlay"></div>
+
+{% block scripts %}
```
Since this template is included, I don't think this block is really used (or as any effect), because block are evaluated before includes (see the green block at the end of the chapter: https://docs.djangoproject.com/en/1.10/ref/templates/builtins/#include). Please review if this file is really being loaded.

dethos commented on hawkpost/templates/account/auth_modals.html:
```diff
@@ -1,29 +1,26 @@
+          <p>If you have not created an account yet, then please<a class=" md-trigger text-darkest" href="#modal-signup" data-modal="modal-signup"><span> sign up </span></a></p>
+          <p class="no-padding">{% get_providers as socialaccount_providers %}
+            {% if socialaccount_providers %}
+          </p>
```
If the above conditional evaluates to false, this tag is never closed.

dethos commented on hawkpost/templates/account/auth_modals.html:
```diff
@@ -35,29 +32,27 @@ <h3 class="title "> Login</h3>
+        <div class="col-xs-12 middle-xs flex-in mt-small">
+          <p class="no-padding">{% blocktrans with site.name as site_name %}Login in via:  {% endblocktrans %}</p>
+          <ul class="gitpad no-margin">
+            {% include "socialaccount/snippets/provider_list.html" with process="login" %}
```
I think this include is dependent on the conditional of line 14 (`{% if socialaccount_providers %}`), no? probably you should also check it here.

dethos commented on hawkpost/templates/account/auth_modals.html:
```diff
@@ -98,15 +88,27 @@ <h3 class="title"> Sign up</h3>
+        <div class="col-xs-12 middle-xs flex-in">
+          <p class="no-padding">Sign up using:</p>
+          <ul class="no-padding">
+            {% include "socialaccount/snippets/provider_list.html" with process="login" %}
```
I think this include is dependent on the conditional of line 72 (`{% if socialaccount_providers %}`), no? probably you should also check it here.

dethos commented on hawkpost/templates/account/auth_modals.html:
```diff
@@ -70,20 +65,15 @@ <h3 class="title "> Login</h3>
         <div class="col-xs-12">
           <h3 class="title"> Sign up</h3>
           {% get_providers as socialaccount_providers %}
           {% if socialaccount_providers %}
```
If doesn't make much sense to check for social account providers in order to put the text `Sign up via email address`, please review this logic on both modals.

dethos reviewed (changes requested):
This pull request needs some changes, the django template logic doesn't makes much sense.

MariCerv committed:
django fixes + box list fixes

- box list close and delete button fixed (now it will only appear in
the open section)
- django fixed

MariCerv committed:
minor fix
